static/js/logs/log.js

```diff
@@ -304,8 +301,12 @@ class Log {
     }
 
     if (this.rawPoints.length === 0) {
-      return {min: 0, max: 1};
+      return {
+        min: this.property.minimum || 0,
+        max: this.property.maximum || 1,
```

Comment:
There's probably a case where maximum could be 0 that would cause this to be incorrect.

Reply:
:+1:, updated accordingly